uaber-api/uaber/routers/trip.py

```diff
@@ -0,0 +1,19 @@
+from fastapi import APIRouter
+
+from uaber.models import Driver, Trip, Refugee, BorderCrossingPoint, DestinationPoint
+from uaber.services import trip as trip_service
+
+router = APIRouter(
+    prefix='/trip',
+    tags=['status'],
+    responses={404: {'description': 'Not found'}}
+)
+
+@router.get('/')
+async def get_drivers():
+    return await trip_service.find_trips()
+
+
+@router.put('/')
+async def creat_trip(driver: Driver, refugee: Refugee, border: BorderCrossingPoint, destination: DestinationPoint) -> Trip:
```

Comment:
I think you should pass `Trip`here, which contains driver, refugee, border and destination already.